src/main/java/no/nav/tag/tiltaksgjennomforing/avtale/Arbeidsgiver.java

```diff
@@ -136,6 +141,12 @@ public boolean harTilgangTilAvtale(Avtale avtale) {
         if (annullertForMerEnn12UkerSiden(avtale)) {
             return false;
         }
+        if (!adressesperreTilgang.contains(avtale.getBedriftNr())) {
+            Diskresjonskode diskresjonskode = persondataService.hentDiskresjonskode(avtale.getDeltakerFnr());
+            if (diskresjonskode.erKode6Eller7()) {
+                return false;
+            }
+        }
```

Comment:
Er ikke dette samme logikk som i `harTilgangPåDeltakerIBedrift` 🤔?

```suggestion
        if (!harTilgangPåDeltakerIBedrift(avtale.getBedriftNr(), avtale.getDeltakerFnr())) {
            return false;
         }
```

Reply:
Bra sett, gikk litt fort her 😅 fikset.